src/main/java/gov/nasa/pds/tools/validate/rule/pds4/ContextProductReferenceValidationRule.java

```diff
@@ -267,39 +266,6 @@ public void checkContextReferences() throws XPathExpressionException {
                     new ProblemDefinition(ExceptionType.INFO, ProblemType.CONTEXT_REFERENCE_FOUND,
                         "Context product found: '" + lidvid + "'"),
                     target, locator.getLineNumber(), -1));
-
-                // now lets check name and type
-                List<String> rgpNames = null;
-                List<String> rgpTypes = null;
-                if (names != null && types != null) {
-                  rgp = rgProds.get(rgProds.indexOf(lidvidObj));
-                  rgpNames = rgp.getNames();
-                  rgpTypes = rgp.getTypes();
-                  // check the name
-                  for (String name : names) {
-                    if (!rgpNames.stream().anyMatch(name::equalsIgnoreCase)) {
-                      getListener().addProblem(new ValidationProblem(
-                          new ProblemDefinition(ExceptionType.INFO,
-                              ProblemType.CONTEXT_REFERENCE_FOUND_MISMATCH,
```

Comment:
@al-niessner This looks like we removed the wrong messages `CONTEXT_REFERENCE_FOUND_MISMATCH`? We still want to check this when we ignore case. I think the problem type we no longer want to catch is `CONTEXT_REFERENCE_FOUND_CASE_MISMATCH`.